client/src/main/java/org/kakara/client/game/world/ClientChunkWriter.java

```diff
@@ -97,39 +87,39 @@ public Chunk getChunkByLocation(ChunkLocation chunkLocation) {
                     KakaraGame.LOGGER.error("Unable to get chunk: " + chunkLocation.toString(), e);
                     //TODO Cancel World Load
                 }
-                ChunkTag chunkTag = null;
-                if (objectTag instanceof ChunkTag) {
-                    chunkTag = (ChunkTag) objectTag;
+                ChunkContentTag chunkContentTag = null;
+                if (objectTag instanceof ChunkContentTag) {
+                    chunkContentTag = (ChunkContentTag) objectTag;
                 } else {
-                    output.add(new NullChunk(chunkLocation));
+                    output.add(new ChunkContent(chunkLocation));
                     continue;
                 }
-                output.add(chunkTag.getChunk());
+                output.add(chunkContentTag.getChunk());
             }
         }
         return output;
     }
 
     @Override
-    public void writeChunk(Chunk chunk) {
+    public void writeChunk(ChunkContent chunk) {
         File chunkFile = getChunkFile(chunk.getLocation());
         ObjectDataStructure ods = new ObjectDataStructure(chunkFile, Compression.GZIP);
         if (ods.find(getChunkKey(chunk.getLocation())))
             ods.delete(getChunkKey(chunk.getLocation()));
```

Comment:
Outdated ODS version. Update to 1.0-BetterMemory and use the set() method.  
Example:
```java
ods.set(getChunkKey(chunk.getLocation()), new ChunkContentTag(chunk));
```